tests/unit/s2n_tls13_handshake_state_machine_test.c

```diff
@@ -208,24 +205,22 @@ int main(int argc, char **argv)
                 EXPECT_NOT_EQUAL(messages_original[j], CLIENT_CHANGE_CIPHER_SPEC);
 
                 /* Skip CCS messages in the MIDDLEBOX_COMPAT handshake */
-                while (messages_mc[j_mc] == SERVER_CHANGE_CIPHER_SPEC ||
-                        messages_mc[j_mc] == CLIENT_CHANGE_CIPHER_SPEC) {
+                while (messages_mc[j_mc] == SERVER_CHANGE_CIPHER_SPEC || messages_mc[j_mc] == CLIENT_CHANGE_CIPHER_SPEC) {
```

Comment:
anyway to enforce line length? here and a few other place.

Reply:
We don't enforce line length. But we can manually add new lines.

Reply:
Two line formatting here is totally possible, you just but the `||` before the second line:

```c
                while (messages_mc[j_mc] == SERVER_CHANGE_CIPHER_SPEC
                        || messages_mc[j_mc] == CLIENT_CHANGE_CIPHER_SPEC) {
```

I'll make this manual change.